recipes/jill/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "jill" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jill-{{ version }}.tar.gz
+    sha256: 48ea86a508f87dab0293d00634b4e8c6677cb00676e617b6c46345bc10af5e10
+  #- url: https://raw.githubusercontent.com/johnnychen94/jill.py/master/requirements.txt
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    ## Ensure inclusion of requirements.txt
+    {% set reqfile = "requirements.txt" %}
+    {% set reqsource = name ~ ".egg-info/requires.txt" %}
+    # IF {{ reqfile }} is NOT PRESENT, COPY {{ reqfile }} from {{ reqsource }}
+    - echo -e "\n>>> ENSURE INCLUSION OF '{{ reqfile }}' <<<\n\n"
+    - if [ ! -f {{ reqfile }} ] && [ -f {{ reqsource }} ]; then cp {{ reqsource }} {{ reqfile }}; fi  # [not win]
+    #- copy {{ name }}.egg-info\requires.txt {{ reqfile }}  # [win]
+    - copy {{ reqsource | replace('/', '\\') }} {{ reqfile }}  # [win]
+    ## Install package
```

Comment:
Why is all of this needed?

Reply:
It is a temporary measure to make the recipe pass the build. I will open an issue at upstream to ask them to include the file in their next release. But, if anything else is missing, would be helpful to know that before I open the issue. Thus can include all of that in a single issue.

Reply:
Ok in that case it's better to switch to GitHub releases (If available) as these scripts don't always work and introduce unnecessary complexity. Conda-forge currently has enough CI time so no problem if the package is slightly bigger.